src/platform/ios/ios.h

```diff
@@ -0,0 +1,12 @@
+#ifndef PLATFORM_IOS_H
+#define PLATFORM_IOS_H
+
+#ifdef __IPHONEOS__
+
+const char* ios_show_c3_path_dialog(int again);
+void c3_path_chosen(char *new_path);
+const char* ios_get_base_path(void);
+int ios_set_base_path(const char *path);
```

Comment:
Be consistent with where you put the `*`. Convention in the project is to add a space between the type and the `*`, also for return values: (also check the other files)
```suggestion
const char *ios_show_c3_path_dialog(int again);
void c3_path_chosen(char *new_path);
const char *ios_get_base_path(void);
int ios_set_base_path(const char *path);
```

Reply:
done!